_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -42,21 +43,24 @@ This rule applies to any set of two or more [HTML or SVG elements][] for which a
 - the elements are in the same [web page (HTML)][]; and
 - the elements are [included in the accessibility tree][included in the accessibility tree]; and
 - the elements have [matching][] [accessible names][accessible name] that are not empty (`""`); and
-- have the same [programmatically determined link context][].
+- the elements have the same [programmatically determined link context][].
 
 **Note:** The test target for this rule is the full set of link elements that share the same [matching][] [accessible name][] and [programmatically determined link context][].
 
 ## Expectation
 
-When followed, the links in each set of target elements resolve to the [same resource][] or to [equivalent resources](#equivalent-resource).
+For each pair of links in each target set, one of the following is true:
+
+- either the links resolve to the [same resource][] or to [equivalent resources](#equivalent-resource); or
+- links resolve to different resources but there is no visual information within the content of the page to predict this behavior.
```

Comment:
```suggestion
- both links resolve to the [same resource][]; or 
- both links resolve to [equivalent resources][]; or
- there is no visual information within the content of the page to let users know that both links resolve to [non-equivalent resources][equivalent resources].
```
Streamlining a bit the list:
* avoid the nested "or";
* start each item with "**both** links" to emphasise the fact that this is about pairs of links;
* remove the pre-condition in the last item since it is an "otherwise" clause for the rest;
* trying to replace "predict" which sounds a bit crystal-bally 🔮 

(there might be need to add `[equivalent resource]: #equivalent-resource 'Definition of Equivalent Resource'` to the references list at the end)